### DISCUSSION TITLE
Duplicated commands

### DISCUSSION THREAD

**Discussion by AMiller42** [Vyxal/Vyxal#820] [Ideas]
This is a list of all commands/overloads that are duplicated. Boxes will be checked when the dupe is resolved, either upon removal, or upon deciding to keep the dupe.

- [ ] `ɾ` (string: a) / `⇧` (string: a) - `a.uppercase()`
- [ ] `ɽ` (string: a) / `⇩` (string: a) - `a.lowercase()`
- [ ] `E` (num: a) / `Ǎ` (num: a) - `2 ** a`
- [ ] `ʁ` (string: a) / `∞` (any: a) / `ṁ` (string: a) - `palindromised a`

**Comment by chunkybanana** [comment]
`ṁ` does *vertical* mirror, `ʁ`'s pointless since`∞` works on lists.

**Comment by chunkybanana** [comment]
`⇧`/`⇩` don't vectorise so `ɾ`/`ɽ` are better sometimes

**Comment by cgccuser** [comment]
Any suggestions for replacing `ʁ`?

**Comment by pxeger** [comment]
`ʁ` could be prefixes including the empty string? (and that could also apply to lists unless we want to keep the vectorisation) That kinda mirrors its behaviour on integers as well

**Comment by AMiller42** [comment]
> `ṁ` does _vertical_ mirror

No, `øṁ` does vertical mirror, whereas `ṁ` does palindromise: [Try it Online!](https://vyxal.pythonanywhere.com/#WyIiLCIiLCJgSGVsbG9cbldvcmxkYOG5gSIsIiIsIiJd)

[Here's `ṁ` in the docs](https://github.com/Vyxal/Vyxal/blob/main/documents/knowledge/elements.md#%E1%B9%81-mean)

Also, `øṁ` doesn't actually appear to do anything, which is an entirely different issue.

**Comment by lyxal** [comment]
I moved this to a discussion so we can plan what should replace what.

I think that `ʁ` on strings should be prefixes like pxeger suggested - it'd be nice to have a vectorising version.

`Ǎ` on numbers could be `a == 1` - it'd be useful for when you have a numeric value you are also treating as a boolean. This'd be different to `ḃ` because `ḃ` considers all non-`0` values truthy, while this would only consider `1` truthy (kind of like how 05AB1E does that by default iirc)
